projects/core/src/lib/models/result.ts

```diff
@@ -0,0 +1,6 @@
+export enum Result {
+	success = "success",
+	gaps = "with some gaps",
+	failed = "failed",
+	comment = "some comment"
```

Comment:
in my opinion more suitable just
	Success,
	Gaps,
	Failed,
	Comment

Reply:
done